titanic.ipynb

```diff
@@ -5,14 +5,14 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 15 to 16_
```python
plt.xlabel('Person Type')
plt.ylabel('Count')
```
Looks good!

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/25)._